modules/cli/src/main/scala/coursier/cli/jvm/Java.scala

```diff
@@ -58,8 +58,13 @@ object Java extends CoursierCommand[JavaOptions] {
                 (name, versionMap) <- map.toVector.sortBy(_._1)
                 version <- versionMap.keysIterator.toVector.map(Version(_)).sorted.map(_.repr)
               } yield s"$name:$version"
-              for (id <- available)
-                System.out.println(id)
+
+              for (id <- available) {
+                val file = jvmCache.getIfInstalled(id).unsafeRun()(coursierCache.ec)
```

Comment:
I guess this is not the right way to compose tasks. But the unique way i can think off is using `sequence` and i am not sure if we have it available

Reply:
Yes, no problem, I pushed an extra commit fixing that.

Reply:
thanks for continuing the pr, a good opportunity for me to learn the api 🙂